### PR TITLE
Rebuild elm value set drc fix

### PR DESCRIPTION
**Pull request by jbradl11**
…ed. It will now grab the value set and the value set version from the existing measure object.

Pull requests into Bonnie Bundler require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1242
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Code coverage has not gone down and all code touched or added is covered.
    * In rare situations, this may not be possible or applicable to a PR. In those situations:
        1. Note why this could not be done or is not applicable here:
        2. Add TODOs in the code noting that it requires a test
        3. Add a JIRA task to add the test and link it here:
- [x] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links: N/A
- [x] Justification for using JIRA tests: N/A
- [x] JIRA tests have been added to sprint N/A

**Reviewer 1:**

Name: @hossenlopp
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

**Comment by mayerm94** [comment]
Worth mentioning that this PR and bonnie PR https://github.com/projecttacoma/bonnie/pull/923 should be tested side by side

File: test/unit/cql_loader_test.rb [hunk 1]
```diff
@@ -51,4 +51,31 @@ class CQLLoaderTest < ActiveSupport::TestCase
...
+      before_value_sets = measure.value_set_oids
+      before_value_set_version_object = measure.value_set_oid_version_objects
+
+      # Re-load the Measure without VSAC Credentials
+      Measures::CqlLoader.load(direct_reference_mat_export, user, measure_details, nil, nil).save
+      assert_equal 2, CqlMeasure.all.count
+      measures = CqlMeasure.all
+
+      # Assert the value sets were loaded properly when no VSAC credentials are provided
+      assert Digest::MD5.hexdigest(before_value_sets.to_json) == Digest::MD5.hexdigest(measures[0].value_set_oids.to_json)
```
Comment (mayerm94) [inline test/unit/cql_loader_test.rb]: 75,77 and 76,78 are redundant. There should be lines asserting that source_data_criteria and data_criteria arrays have not changed either

**Review by hossenlopp — APPROVED** [review]
Expected travis failure due to nokogiri. But otherwise, looks good!

**Review by mayerm94 — APPROVED** [review]
Changes since last review look good.
